src/components/modes/student/widgets/components/SpeedOMeter.js

```diff
@@ -41,7 +44,7 @@ const SpeedOMeter = ({ value, colors, segments }) => {
           />
         </div>
         <Typography variant="body1" display="block" gutterBottom>
-          {`Your participation: ${value[USER]}`}
+          {`Your participation: ${value[USER] || 0}`}
```

Comment:
Do we actually have `t()` setup here? If not, just leave it (and ignore previous comment).